src/middlewared/middlewared/plugins/chart_releases_linux/utils.py

```diff
@@ -62,6 +62,42 @@ def get_chart_release_from_namespace(namespace):
     return namespace.split(CHART_NAMESPACE_PREFIX, 1)[-1]
 
 
+def safe_to_ignore_path(path: str) -> bool:
+    # "/" and "/home/keys/" are added for openebs use only, regular containers can't mount "/" as we have validation
+    # already in place by docker elsewhere to prevent that from happening
+    if path == '/':
+        return True
+
+    for ignore_path in (
+        '/etc/',
+        '/sys/',
+        '/proc/',
```

Comment:
I think we need to be quite careful about whether we allow /proc access. It is very easily abused.

Reply:
I think this is just a copy-paste from the current validator.

However, you're 100% correct... The validation should (aim to) prevent both user-error and potential container escapes (which is a known issue with hostPaths on kubernetes)

I don't see a usecases where it's really required to have host-level access here.